sqoop/scripts/rucio_ds_daily_stats.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+set -e
+# Reference: some features&logics are copied from sqoop_utils.sh and cms-dbs3-full-copy.sh
```

Comment:
@mrceyhun please add proper documentation to the beginning of this script. It should include:
- all options script can use
- all hard-coded areas, like /etc/secrets/rucio and /data/sqoop/cmsr_cstring as well as their meaning and how they should be setup and/or used
- all env variables you rely on, I see some them use absolute path but other relative ones. I don't know if you check that relative ares, like log, actually exists before using it
- on exit it should print location of the logs to speed up debugging process, in fact if we'll run it on k8s it is better to dump log files on stdout such that we can use kubectl log of the pod without logging to the pod.

Reply:
thanks @vkuznet , I added checks and documentation. My initial idea is to run it in sqoop pod as other sqoop import jobs. Its cron is added to `cronjobs.txt`. 

If I successfully send Spark results to MONIT via Stomp and Rucio colleagues find it useful, maybe we can create a K8s service that runs sqoop import and the Spark job sequentially.